docs/source/documentation/patterns/sample-code/_accordions.html

```diff
@@ -1,9 +1,7 @@
 <aside class="js-accordion accordion {{modifier}}">
   <div class="accordion-section">
     <h4 class="accordion-title">
-    <span class="accordion-icon">
-      <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 32 32" class="svg-icon"><path d="M28 9v5L16 26 4 14V9l12 12L28 9z"/></svg>
-    </span>
+    {% icon 'chevron-right', size=16, classes="accordion-icon" %}
```

Comment:
I messed up, these should be `chevron-down` I think

Reply:
fixed